ensembl-track-api.openapi.yaml

```diff
@@ -135,7 +135,7 @@ components:
         track_categories:
           type: array
           items:
-            $ref: '#/components/schemas/TrackCategory'
+            $ref: '#/components/schemas/CategoryTracks'
```

Comment:
This looks odd. How come `TrackCategories` (line 132) is not a plural form of `TrackCategory`?

Reply:
There are 2 forms of TrackCategory:
- `TrackCategory`: Track category object itself (used in `POST /track` payload)
- `CategoryTracks`: Track category object with nested Track list (used here)
Alternative names like `TrackCategoryWithTrackList` seemed too long.

Reply:
`TrackCategoryWithTracks`?

But I am not seeing any track category object described in POST /track payload 🤷‍♂️

Reply:
I considered `CategoryTracks` a variant of `Tracks` (list of tracks grouped to categories), but I see why it's confusing. I'll update the name. `TrackCategory` is part of `Track` schema.